PyTorch_dcgan_lsun/README.md

```diff
@@ -0,0 +1,64 @@
+# PyTorch - DCGAN
+
+This example is based on [this implementation](https://github.com/pytorch/examples/tree/master/dcgan)
+```
+git clone https://github.com/pytorch/examples.git
+```
+
+## Downloading the dataset
+
+You can download the LSUN dataset by cloning [this repo](https://github.com/fyu/lsun) 
+
+```
+git cllone https://github.com/fyu/lsun.git
+```
+and running
+```
+python download.py -c bedroom
+```
+
+For testing and/or benchmarking purposes it is also possible to the train the model on fake dataset. To do this run this command instead of the one given in the Usage section:
+
+```
+CUDA_VISIBLE_DEVICES=0 python main.py --init-method tcp://127.0.0.1:20011 --rank 0 --world 2 --dataset fake --cuda
+CUDA_VISIBLE_DEVICES=1 python main.py --init-method tcp://127.0.0.1:20011 --rank 1 --world 2 --dataset fake --cuda
+```
```

Comment:
Remove the commands here and leave them only in the further section?

Reply:
I think it is easier for the user to copy whole command than to just change a part of it

Reply:
You're right, but on the other hand:
- we put these modified commands before the regular ones are introduces, which may be confusing
- the user will have to modify the commands to suit their environment anyway
It's your call then.